bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
I just don't see that.

"https://bugs.eclipse.org/bugs/buglist.cgi?last_change_time=2016-01-27%2010%3A00&list_id=13711404&status_whiteboard=RHT" works

"https://bugs.eclipse.org/bugs/buglist.cgi?chfieldfrom=2016-01-27%2010%3A00&list_id=13711405&status_whiteboard=RHT" does not.
Also your code already calculate the difference so chfieldfrom is not even used/needed - nor are they the same thing. one is a date the other is a relative time.

so i'm not following why this change is needed.